parachain/runtime/kintsugi/src/lib.rs

```diff
@@ -263,6 +263,26 @@ parameter_types! {
     pub const ExecutiveBody: BodyId = BodyId::Executive;
 }
 
+// Set the initial parameters for CollatorSelection.
+pub struct SetCollatorSelection;
+impl frame_support::traits::OnRuntimeUpgrade for SetCollatorSelection {
+    fn on_runtime_upgrade() -> frame_support::weights::Weight {
+        collator_selection::DesiredCandidates::<Runtime>::put(9);
+        collator_selection::CandidacyBond::<Runtime>::put(600 * KINT.one());
```

Comment:
Can you please make this a parameter that can be set by governance so that we don't need a runtime upgrade to change this?

Reply:
They are already updateable by governance, but since the pallet changed we would need to wait to propose those until after the upgrade. The idea behind setting these now is to include the proposed values in the same governance upgrade (taking 7d instead of 7d + 7d).

Reply:
Makes sense, thanks!